src/pages/Environments.js

```diff
@@ -457,7 +457,7 @@ const Environments = () => {
             field: 'workspace',
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'workspace', onSort: setDiskSort }, ['Workspace']),
             cellRenderer: ({ rowIndex }) => {
-              const { status: diskStatus, googleProject, labels: { saturnWorkspaceNamespace, saturnWorkspaceName } } = filteredDisks[rowIndex]
+              const { status: diskStatus, googleProject, labels: { saturnWorkspaceNamespace = googleProject, saturnWorkspaceName } } = filteredDisks[rowIndex]
```

Comment:
Assuming this just populates a default value if saturnWorkspaceNamespace doesn't exist (I'm still a newbie at this syntax), this LGTM.

Reply:
Exactly.